### PR TITLE
Support multiple ingresses; add Drone lint

### PR DESCRIPTION
**Pull request by zuzak**

### CODE REVIEW

**Comment by zuzak** [comment]
Looking back at this, the equivalent frontend change (https://github.com/UKHomeOffice/kube-hocs-frontend/pull/86/files) tells Keycloak the various redirect URLs explicitly in demo and above. We'll need to do that here too, or the logout links will break

**Review by Paulcr-ho — APPROVED** [review]

File: deploy.sh [hunk 1]
```diff
@@ -1,77 +1,57 @@
...
+if [[ ${SUBNAMESPACE} == "prod" ]]
 then
     export MIN_REPLICAS="2"
     export MAX_REPLICAS="6"
     export KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
     export UPTIME_PERIOD="Mon-Sun 05:00-23:00 Europe/London"
+    export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-prod.crt"
+    if [[ "${DOMAIN}" == "wcs" ]] ; then
+        export EXTERNAL_DOMAIN=www.wcs-management.homeoffice.gov.uk
+        export INTERNAL_DOMAIN=''
```
Comment (LiamMacP) [inline deploy.sh:24]: This can be pulled out outside of this `if` block. Repeated on L28.

Comment (zuzak) [reply deploy.sh:24]: It could, but I think I'd rather repeat them twice for clarity.

Comment (LiamMacP) [reply deploy.sh:24]: Is there really any clarity needed – we set everything else in the prod block so why not this?

Comment (zuzak) [reply deploy.sh:24]: It's nice to have the internal and external ingress declarations next to each other. I think it's semantically valid: they're highly related variables that in more normal languages might be an array.

**Review by LiamMacP — APPROVED** [review]

File: deploy.sh [hunk 2]
```diff
@@ -1,77 +1,57 @@
...
 
-if [[ ${KUBE_NAMESPACE} == *prod ]]
+export SUBNAMESPACE="${KUBE_NAMESPACE#*-}" # e.g. dev, qa
+
+if [[ ${SUBNAMESPACE} == "prod" ]]
 then
     export MIN_REPLICAS="2"
     export MAX_REPLICAS="6"
     export KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
     export UPTIME_PERIOD="Mon-Sun 05:00-23:00 Europe/London"
```
Comment (Paulcr-ho) [inline deploy.sh:20]:
```suggestion
    export UPTIME_PERIOD="Mon-Sun 05:00-23:00 Europe/London"
    export INTERNAL_DOMAIN=''    
```

File: deploy.sh (hunk 1 above)
Comment (Paulcr-ho) [reply deploy.sh:24]:
```suggestion
```

File: deploy.sh [hunk 3]
```diff
@@ -1,77 +1,57 @@
...
+    export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-prod.crt"
+    if [[ "${DOMAIN}" == "wcs" ]] ; then
+        export EXTERNAL_DOMAIN=www.wcs-management.homeoffice.gov.uk
+        export INTERNAL_DOMAIN=''
+        export KC_REALM=https://sso.digital.homeoffice.gov.uk/auth/realms/HOCS
+      else
+        export EXTERNAL_DOMAIN=www.cs-management.homeoffice.gov.uk
+        export INTERNAL_DOMAIN=''
```
Comment (Paulcr-ho) [inline deploy.sh:28]:
```suggestion
```

**Pull request merged**